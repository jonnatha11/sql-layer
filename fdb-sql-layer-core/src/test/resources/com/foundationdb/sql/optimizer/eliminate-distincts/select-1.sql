SELECT DISTINCT name FROM parent
