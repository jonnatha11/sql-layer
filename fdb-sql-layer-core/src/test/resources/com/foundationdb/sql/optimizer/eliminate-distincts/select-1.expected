SELECT test.parent.name AS name FROM test.parent